include/pmacc/math/ConstVector.hpp

```diff
@@ -87,15 +87,15 @@
         } /* namespace pmacc_static_const_vector_host + id  */                                                        \
         /* select host or device namespace depending on __CUDA_ARCH__ compiler flag*/                                 \
         PMACC_USING_STATIC_CONST_VECTOR_NAMESPACE(id);                                                                \
-        template<typename T_Type, int T_Dim>                                                                          \
+        template<typename T_Type, uint32_t T_Dim>                                                                     \
```

Comment:
I am not exactly sure why you started to rename `T_Dim` -> `T_dim` at some places, but I guess it is supposed to be consistent?
You missed to rename here.

(Although I would have expected that `T_Dim` actually complies w/ our standard naming scheme and not `T_dim`)

Reply:
If the dimension would be a type then the naming `T_Dim` would be fine. In this case, it is a value and should be `T_dim`. I will change it!